src/components/video/video.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class Video extends React.Component {
+  render() {
+    let videoProps = {
+      autoPlay: true,
+      loop: true
+    };
+    const mediaQuery = window.matchMedia('(prefers-reduced-motion: reduce)');
```

Comment:
You'll want to do a slight swap and make this line evaluate if `window` exists, rather than the next line.